sematic/db/migrations/20230428225443_rename_calculator_path.sql

```diff
@@ -0,0 +1,7 @@
+-- migrate:up
+
+ALTER TABLE runs RENAME COLUMN calculator_path TO function_path;
```

Comment:
Please validate that rerun-from-here still works with original pipelines pre-rename.